model/simulationtests/heatexchanger_desiccant_balancedflow.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+# make a 2 story, 100m X 50m, 10 zone core/perimeter building
+model.add_geometry({ 'length' => 100,
+                     'width' => 50,
+                     'num_floors' => 1,
+                     'floor_to_floor_height' => 4,
+                     'plenum_height' => 1,
+                     'perimeter_zone_depth' => 3 })
+
+# add windows at a 40% window-to-wall ratio
+model.add_windows({ 'wwr' => 0.4,
+                    'offset' => 1,
+                    'application_type' => 'Above Floor' })
+
+# add thermostats
+model.add_thermostats({ 'heating_setpoint' => 24,
+                        'cooling_setpoint' => 28 })
+
+# assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions
+
+# set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type
+
+# add design days to the model (Chicago)
+model.add_design_days
+
+# add ASHRAE System type 03, PSZ-AC
+model.add_hvac({ 'ashrae_sys_num' => '03' })
+
+air_systems = model.getAirLoopHVACs
+
+air_systems.each_with_index do |s, i|
+  oa_node = s.airLoopHVACOutdoorAirSystem.get.outboardOANode.get
+
+  if i == 0
+    p = OpenStudio::Model::HeatExchangerDesiccantBalancedFlowPerformanceDataType1.new(model)
+    hx = OpenStudio::Model::HeatExchangerDesiccantBalancedFlow.new(model, p)
```

Comment:
Please exercise ALL setters for both objects at least once (you can copy all defaults from the model Ctor and just prepend with `hx.` or `.p` as appropriate, that is fine). The goal is both to test the object but also to demonstrate how to use its api. Also, harcoding values in the ruby test means that should we adjust the default in model ctor, we wouldn't see a deviation in EUI suddenly.